node/service/src/chain_spec.rs

```diff
@@ -274,6 +274,44 @@ pub fn cere_dev_development_config() -> Result<CereDevChainSpec, String> {
 	))
 }
 
+#[cfg(feature = "cere-dev-native")]
+fn cere_dev_local_testnet_genesis(wasm_binary: &[u8]) -> cere_dev::GenesisConfig {
+	cere_dev_genesis(
+		wasm_binary,
+		// Initial authorities
+		vec![authority_keys_from_seed("Alice"), authority_keys_from_seed("Bob")],
+		// Initial nominators
+		vec![],
+		// Sudo account
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		// Pre-funded accounts
+		Some(vec![
+			get_account_id_from_seed::<sr25519::Public>("Alice"),
+			get_account_id_from_seed::<sr25519::Public>("Bob"),
+			get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+			get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+		]),
+	)
+}
+
+#[cfg(feature = "cere-dev-native")]
+pub fn cere_dev_local_testnet_config() -> Result<CereDevChainSpec, String> {
+	let wasm_binary = cere_dev::WASM_BINARY.ok_or("Cere Dev development wasm not available")?;
+
+	Ok(CereDevChainSpec::from_genesis(
+		"Local Testnet",
+		"local_testnet",
```

Comment:
Keep in mind, that in case of `local_testnet` value for the id field **you will not be able to run the node with the cere-dev runtime** that contains experimental pallets such as dacValidator, dacAccounts, etc

Reply:
Fixed.